userguide/image_classifier/README.md

```diff
@@ -30,6 +30,43 @@ data.save('cats-dogs.sframe')
 data.explore()
 ```
 
+#### Annotating Data
+
+If you only have images without corresponding annotations, you can 
```

Comment:
I would suggest using the word `labels` rather than `annotations` in the first sentence, to help introduce the context.